.github/workflows/tag_action.yml

```diff
@@ -0,0 +1,40 @@
+name: Report tag to Notion
+
+on:
+  push:
+    tags:
+      - rc*
+env:
+  NOTION_DATABASE_ID: e3aca840563b4cc8b273e65da748e1dd
+  WEBHOOK_URL: https://hook.eu2.make.com/sg1xhhlvnp4314llgde92csqgjrpm42e
+
+jobs:
+  report_tag_to_notion:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Set up environment variables
+        run: |
+          echo "NOTION_DATABASE_ID=e3aca840563b4cc8b273e65da748e1dd" >> $GITHUB_ENV
+          echo "WEBHOOK_URL=https://hook.eu2.make.com/sg1xhhlvnp4314llgde92csqgjrpm42e" >> $GITHUB_ENV
```

Comment:
We have these definitions in the `env:` section at the top already. Can be removed?

Reply:
removed